tests/unit/containers/tutor-home-page/general-info-step/GeneralInfoStep.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import { cleanup, render, screen } from '@testing-library/react'
+import GeneralInfoStep from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep'
+
+describe('GeneralInfoStep test', () => {
+  const mockButton = <button>Mock Button</button>
+
+  beforeEach(() => {
+    render(<GeneralInfoStep btnsBox={mockButton} />)
+  })
+
+  afterEach(() => {
+    cleanup()
+  })
+
+  it('Renders GeneralInfoStep container', () => {
+    const containerElement = screen.getByText(/GeneralInfo step/i)
+    expect(containerElement).toBeInTheDocument()
+  })
+
+  it('The buttons passed in props is in the document', () => {
```

Comment:
The description of the test should sound as a continuation of the `it`. Something like `it(should render the buttons...)`. Read [here](https://daily-dev-tips.com/posts/jest-test-vs-it/) more.

Reply:
Okay, didn't notice that, thanks!